BoxLEDs.cpp

```diff
@@ -40,16 +78,141 @@ void BoxLEDs::disableRedLED(bool disabled) {
     }
 }
 
-void BoxLEDs:: setIdleType(IDLE_TYPE idleType) {
-    _idleType = idleType;
+void BoxLEDs::setIdleAnimation(ANIMATION_TYPE animationType,ANIMATION_COLOR animationColor) {
```

Comment:
Code duplication with setActiveAnimation.
A private helper function could use a pointer to generalize the code

Reply:
Done in commit 640ca076236cc24a14c615ce5628263df4c6d060
private helper function addAnimation to remove code duplication